foruma-frontend/src/stories/Button.vue

```diff
@@ -0,0 +1,52 @@
+<template>
+  <button type="button" :class="classes" @click="onClick" :style="style">{{ label }}</button>
+</template>
+
+<script>
+import './button.css';
+import { reactive, computed } from 'vue';
+
+export default {
+  name: 'my-button',
```

Comment:
Codacy has a fix for the issue: [Replace `'my-button'` with `"my-button"`](https://app.codacy.com/gh/agabani/foruma/pullRequest?prid=7388752)
```suggestion
  name: "my-button",
```